python/py-allennlp/Portfile

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        allenai allennlp a8c44ae
```

Comment:
don't specify a commit-hash, use a proper version instead

Reply:
The latest official version is behind on its dependencies, so 0.9.1-prerelease is the best option right now. 

I’m not 100% convinced that it’s ready either, but am mostly convinced based on my own usage. It’ll take a while to eat through these edits and get a working build, so perhaps official 0.9.1 will re ready by then.

Until then it has to be the latest github hash.

Reply:
I’m pretty sure the latest stable upstream release does work - correct? So why not package that for now, you should not package development versions unless in a -devel port.

 I am not sure I understand - what does your statement “is behind on its dependencies” mean?

Reply:
See https://github.com/allenai/allennlp/issues/3481

Latest official doesn’t work with transformers, other dependencies. It can be modded to work, though.